brage-import/src/main/java/no/sikt/nva/brage/migration/merger/findexistingpublication/HandleFinder.java

```diff
@@ -15,9 +16,12 @@
 public class HandleFinder implements FindExistingPublicationService {
 
     private final SearchApiFinder searchApiFinder;
+    private final DuplicatePublicationReporter duplicatePublicationReporter;
 
-    public HandleFinder(ResourceService resourceService, UriRetriever uriRetriever, String apiHost) {
+    public HandleFinder(ResourceService resourceService, UriRetriever uriRetriever, String apiHost,
+                        DuplicatePublicationReporter duplicatePublicationReporter) {
         this.searchApiFinder = new SearchApiFinder(resourceService, uriRetriever, apiHost);
+        this.duplicatePublicationReporter = duplicatePublicationReporter;
     }
 
     @Override
```

Comment:
It seems like HandleFinder is the only finder that does not filter out publications as others finders do by this method/ class:

`.filter(item -> PublicationComparator.publicationsMatch(item, publicationRepresentation.publication()))`

Is there a special handling of publications fetched by handle? If no, it could also be nice to somehow move this method to the findExistingPublicationService, so we always "filter out" not relevant publications by default.

Reply:
The only reason there is a handle in additionalIdentifier is because a brage-migration has previously occurred.
So either we want to update all the values (probably), or discard the entire thing.